src/css/site.styl

```diff
@@ -48,6 +52,25 @@ img
 .desktop-only
   display: none !important
 
+input, select, a
+  &:focus
+    outline: none
+    color: $color-red
+    img
+      box-shadow: 0 0 40px -5px $color-red
```

Comment:
doesnt need to be now but seems like this could be a mixin for use on other things?